### ISSUE TITLE
octave info beside the note 

### ISSUE THREAD

**Issue by sugizo** [cheap-glitch/fretboarder#18]
#### Description

better to have octave info beside the note useful when play in choir 
e.g. in standard choir usually use SATB (Sopran Alto Tenor Bass) which every vocal characteristic (male/female) has it's own range (base on octave)

#### Implementation details

perhaps use button to show octave info for show or hidden the info (recommended because not every guitarist care about octave can hide it, but for the detail guitarist that must play within choir can show the octave)
or  
simply put octave info without option to show or hide octave info

#### Examples

which fretboard must be press to play melody with tenor vocal on major third tuning

![tenor](https://user-images.githubusercontent.com/2418295/93690506-5fdcc900-fb03-11ea-8ca2-0cfa778429a6.png)

**ref on piano**
https://en.wikipedia.org/wiki/Tenor

thanks

**Comment by cheap-glitch** [comment]
I agree this could be useful, but it's going to be quite a bit of work to support this, as currently the music-related logic works on "abstract" notes and doesn't take octaves into account at all.

I'll keep this in mind for a possible future major version bump.

**Comment by sugizo** [comment]
perhaps can learn from this 
http://sugizo.pythonanywhere.com/music/fretboard/form_scale_in_fretboard
created using python and could share the code if you want
